src/bootstrap/bootstrap.py

```diff
@@ -1004,6 +1004,14 @@ def bootstrap(help_triggered):
         with open(toml_path) as config:
             build.config_toml = config.read()
 
+    profile = build.get_toml('profile')
+    if profile is not None:
+        include_file = 'config.{}.toml'.format(profile)
+        include_dir = os.path.join(build.rust_root, 'src', 'bootstrap', 'defaults')
+        include_path = os.path.join(include_dir, include_file)
+        with open(include_path) as included_toml:
+            build.config_toml += '\n' + included_toml.read()
```

Comment:
I'm 90% sure this will break - this causes invalid syntax if there's already anything in the config.toml. https://github.com/rust-lang/rust/pull/76628#issuecomment-695157956

Reply:
It would break if parsed by a real TOML parser, but [that's not the case](https://github.com/rust-lang/rust/blob/d65c08e9cc164b7b44de53503fae859a4fafd976/src/bootstrap/bootstrap.py#L656), the basic regex parser will just match the first relevant key